discovery-provider/Caddyfile

```diff
@@ -15,4 +15,4 @@ route /dashboard* {
     file_server
 }
 
-reverse_proxy backend:5000
+reverse_proxy metabase:3000
```

Comment:
@alecsavvy since you are here. wdyt about having this be configurable? it seems like the move would just be to allow a dev to add an override.env var and have that be used as the main entrypoint

Reply:
I'm not too familiar with how caddy reads in env files but yeah. Looks like you can!
Wherever we do the caddy run command we should be able to do something like `caddy run --envfile override.env`. From there we could read in like `metabase:{$METABASE_PORT}` and subsequently pass it into the docker compose file like we do acdc here https://github.com/AudiusProject/audius-docker-compose/blob/stage/discovery-provider/docker-compose.yml#L317.